.yardstick.yml

```diff
@@ -0,0 +1,14 @@
+---
+rules:
+  ApiTag::Presence:
+    enabled: true
+    exclude:
+      - Guard::YardstickVersion.to_s
```

Comment:
Broke a few rules around the code to ensure configuration continues to work going forward.